src/sections/home/Home.Events.tsx

```diff
@@ -15,21 +15,24 @@ const HomeEvents: React.FC<{ events: Array<EventObject> }> = ({ events }) => {
         {events && events.length > 0 ? (
           events.map((value, index) => {
             const timing = getDateTime(value);
+            const eventLink = "https://acmucsd.com/events/" + value.title.replace(/ /g, "-") + "-" + value.uuid;
```

Comment:
Linking directly to acmucsd.com might have some weird consequences when testing locally. For example, if you're running the website locally and using the testing API, it might create an event card with a link that doesn't exist on the main ACMUCSD website, when it should link you to your localhost with the testing event instead.

I think the better convention here is to link to `"/events/{uuid}" instead of "https://acmucsd.com/events/{uuid}" since this is a relative link instead of an absolute one.

Reply:
Agree with Alex's comment - it made sense to link to acmucsd.com during the Diamond Staff practice project because we were on a completely separate website however the page link we are trying to redirect to is actually in our same `main-website` repo now so we want to use relative link paths so that we stay on our same website regardless of what the actual website URL is. This way the link will work from `localhost:3000` to `localhost:3000/event/{uuid}` and `acmucsd.com` to `acmucsd.com/event/{uuid}` exactly the same. 

Two other comments on this line of code:

1. We have an existing helper function to consistently format the title portion of the URL in the same way always called `formatURLEventTitle` that you can import from `src/utils/general`

![Screenshot 2023-03-30 at 9 25 46 PM](https://user-images.githubusercontent.com/33165426/229022885-65ee1c54-ade9-4ce5-b8c5-bb5d5a95cdcc.jpg)
2. JavaScript has a feature called template literals that let us concatenate strings with variables in a much more readable fashion that is white-space sensitive so we get the exact string we want.

e.g. Instead of this,
```JavaScript
const name = "Carmen";
console.log("Hello " + name); // "Hello Carmen"
```
we would write this in backticks.

```JavaScript
const name = "Carmen";
console.log(`Hello ${name}`); // "Hello Carmen"
```